classes/class-wc-connect-compatibility-wc26.php

```diff
@@ -116,6 +116,22 @@ public function get_product_name_from_order( $product_id, $order ) {
 			return sprintf( __( '#%d - [Deleted product]', 'woocommerce-services' ), $product_id );
 		}
 
+		/**
+		 * For a given product ID, it tries to find its price inside an order's line items.
+		 *
+		 * @param int $product_id Product ID or variation ID
+		 * @param WC_Order $order
+		 * @return float The product (or variation) price, or NULL if it wasn't found
+		 */
+		public function get_product_price_from_order( $product_id, $order ) {
+			foreach ( $order->get_items() as $line_item ) {
+				if ( (int) $line_item[ 'product_id' ] === $product_id || (int) $line_item[ 'variation_id' ] === $product_id ) {
+					return round( floatval( $line_item[ 'total' ] ) / $line_item[ 'qty' ], 2 );
+				}
+			}
+			return null;
+		}
+
```

Comment:
I think we agreed to stop supporting WC 2.x after 3.4 was released?

Reply:
Yes, we did (I started this before 3.4 was released haha).

Do you think we should stop supporting it right now, or should we do a release where we explicitly remove support? (May be reasonable to do a `2.0.0` version in that case).

Reply:
Fine by me to isolate the back-compat changes. 👍